spec/unit/backup_spec.rb

```diff
@@ -1,40 +1,42 @@
 require_relative 'spec_helper'
+# rubocop:disable ModuleLength
```

Comment:
you shouldn't introduce new offenses while solve an offense... :disappointed:

Reply:
The module size is too large according to Rubocop. I thought the code logic needs to be in one file that's why I didn't split them.
In order to do that, I think I need to split this big module in small ones? Or we can do something as below in `rubocop.yml` :
`
ClassLength:
  Max: 250
`
 What you think on that?

Reply:
you should not introduce new offenses, neither change the configuration. If a module is too long, some thing needs to be out of the module. But how you are fixing the `Style/MixinUsage` offenses is not right. Maybe you don't need to touch this, focus on solving `Style/MixinUsage` first. :wink: